values.yaml

```diff
@@ -5,6 +5,14 @@
 
 captain_domain: nil
 
+glueops_backups:
+  vault:
+    aws_region: nil
+    aws_accessKey: nil
+    aws_secretKey: nil
+    cron_expression: "0 */6 * * *"
```

Comment:
confirming we want to run this 4x / day

Reply:
Yes. Figure we could change it later but since the backups are so small it shouldn't hurt to have more